File: public_html/cgi-bin/Create.java
import java.sql.*;
import java.io.*;
import oracle.jdbc.*;
import oracle.jdbc.pool.OracleDataSource;
import java.util.Arrays;
import java.util.stream.Stream;

public class Create {
    public static void main(String[] args) throws SQLException{
        String user     = "C##benjamin.stafford";
        String password = "stafford6248";
        String database = "65.52.222.73:1521/cdb1";
        OracleDataSource ods = new OracleDataSource();
        ods.setURL("jdbc:oracle:thin:@" + database);
        ods.setUser(user);
        ods.setPassword(password);
        Connection conn = ods.getConnection();
        try {
            Statement stmt = conn.createStatement();
            String query = "insert into (title, author, isbn, subjects) values (";
            Arrays.stream(args).forEach(x -> query += x.trim() + " ");
            query += ");";
            ResultSet rset = stmt.executeQuery(query);
            System.out.println("Content-Type: text/javascript");
            System.out.println("alert(\"You submitted " + query + "\");");
            rset.close();
            stmt.close();
        }
        catch (SQLException ex) {
            System.out.println(ex);
        }
        conn.close( );
    }
}

File: public_html/cgi-bin/create.pl
#!/usr/bin/perl
use CGI; 
$query = new CGI;
$title = $query->param('book-title');
$author = $query->param('price');
$isbn = $query->param('isbn');
$subjects = $query->param('subjects');

$title =~ s/^\s*(\S*)\s*$/$1/;
$title =~ s/;|>|>>|<|\*|\?|\&|\|//g;
$author =~ s/^\s*(\S*)\s*$/$1/;
$author =~ s/;|>|>>|<|\*|\?|\&|\|//g;
$isbn =~ s/^\s*(\S*)\s*$/$1/;
$isbn =~ s/;|>|>>|<|\*|\?|\&|\|//g;
$subjects =~ s/^\s*(\S*)\s*$/$1/;
$subjects =~ s/;|>|>>|<|\*|\?|\&|\|//g;

$cmd = "/usr/bin/java -Djava.security.egd=file:/dev/./urandom Create ";
$cmd .= join $title, " ", $author, " ", $isbn, " ", $subjects;

system($cmd);